ajax/libs/heapbox/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "heapbox",
+  "version": "0.9.4",
+  "filename": "jquery.heapbox-0.9.4.min.js",
```

Comment:
I'll wait for the author response to several days.(Maybe 5 to 7 days)
If the author didn't any answer, I'll add them manually, and remove the auto-update part from `package.json`.
Thanks.